includes/class-wc-gateway-stripe.php

```diff
@@ -1183,4 +1170,36 @@ public function process_admin_options() {
 			update_option( 'wc_stripe_show_changed_keys_notice', 'yes' );
 		}
 	}
+
+	public function validate_publishable_key_field( $key, $value ) {
+		$value = $this->validate_text_field( $key, $value );
+		if ( ! empty( $value ) && ! preg_match( '/^pk_live_/', $value ) ) {
+			throw new Exception( __( 'The "Live Publishable Key" should start with "pk_live", please make sure you have entered the correct key.', 'woocommerce-gateway-stripe' ) );
```

Comment:
"make sure you have entered the correct key" -> "enter the correct key" here and below

Reply:
addressed in 31aec10dde7a978a3d47e60b5b8f4a4a56a6e8d9